Update llms.md (#507)
citynetwork/docs: docs/reference/ai/llms.md

> {{brand_ai}} currently offers five on-demand models.

Reviewer: ```suggestion
{{brand_ai}} currently offers six on-demand models.
```

Refer to the table below for information about each model's origin, and their relative strengths.

| Model                   | Developed by                  | Designed for                                                                                  |
| ----------------------- | ----------------------------- | --------------------------------------------------------------------------------------------- |
| faster-whisper-large-v3 | OpenAI                        | Speech recognition and translation                                                            |

> | kb-whisper-large        | Kungliga Biblioteket & OpenAI | Speech recognition and translation (fine-tuned for transcription accuracy of Nordic languages |

Reviewer: ```suggestion
| kb-whisper-large        | Kungliga Biblioteket & OpenAI | Speech recognition and translation (fine-tuned for transcription accuracy of Nordic languages) |
```

{{brand_ai}} currently offers five on-demand models.
Refer to the table below for information about each model's origin, and their relative strengths.

| Model                   | Developed by                  | Designed for                                        |
| ----                    | ----------------------------  | --------------------------------------------------- |
| faster-whisper-large-v3 | OpenAI                        | Speech recognition and translation                  |

> | kb-whisper-large        | Kungliga Biblioteket & OpenAI | Improved transcription accuracy of Nordic languages |

Reviewer: ```suggestion
| kb-whisper-large        | Kungliga Biblioteket & OpenAI | Speech recognition and translation (fine-tuned for transcription accuracy of Nordic languages |
```

> | gemma-3-27b \*          | Google                        | Routine natural language processing tasks           |

Reviewer: Please don't use random asterisks in tables. If absolutely necessary, use footnotes, but in this case it's fine to skip those, too.

| gemma-4-31b-it-fp8      | Google                        | Complex reasoning, multimodal, agentic tasks        |
| gpt-oss-120b            | OpenAI                        | Complex reasoning                                   |
| qwen3-coder-30b \*      | Alibaba Cloud                 | Code generation, agentic tasks                      |
| qwen3-embedding-8b      | Alibaba Cloud                 | Text embeddings, ranking                            |
| qwen3.6-35b-a3b-fp8     | Alibaba Cloud                 | Code generation, agentic tasks                      |

> _gemma-3-27b and qwen3-coder-30b are scheduled to go offline 2026-06-04 00:00 CET._

Reviewer: This is best expressed in an admonition. Also, when giving dates with a time component, use UTC for the latter.
